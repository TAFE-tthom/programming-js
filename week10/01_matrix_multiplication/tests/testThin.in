1 5 10 1
2
5
3
0
11
3 4 19 4 1 0 11 3 7 7
