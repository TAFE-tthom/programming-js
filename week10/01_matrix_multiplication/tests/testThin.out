6 8 38 8 2 0 22 6 14 14
15 20 95 20 5 0 55 15 35 35
9 12 57 12 3 0 33 9 21 21
0 0 0 0 0 0 0 0 0 0
33 44 209 44 11 0 121 33 77 77
